includes/class-sensei-admin.php

```diff
@@ -202,27 +199,14 @@ public function page_redirect() {
 	 *
 	 * Handles installation of the 2 pages needs for courses and my courses
 	 *
+	 * @deprecated since 3.1.0 use Sensei_Onboarding_Pages::create_pages
```

Comment:
Comparing with the latest code, I think the standard should be:
``@deprecated 3.1.0 Use `Sensei_Onboarding_Pages::create_pages` instead``

Also for the others in this PR.

Reply:
Updated the comments.